include/emitc/emitc_tosa.h

```diff
@@ -178,6 +178,91 @@ Dest conv2d(Src input, Weights weights, Tensor1D<int64_t, 4> padding,
   return output;
 }
 
+// DepthwiseConv2DOp
+template <typename Dest, typename Src, typename Weights>
+Dest depthwise_conv2d(Src input, Weights weights, Tensor1D<int64_t, 4> padding,
+                      Tensor1D<int64_t, 2> stride,
+                      Tensor1D<int64_t, 2> dilation) {
+  // Input is [N,IH,IW,IC], weights
```

Comment:
Why are you not using the same notation as the constants in the function itself?

`IH` -> `H_IN` e.g.

Reply:
Yep, I agree: that's confusing.